<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Eingabefenster</class>
 <widget class="QMainWindow" name="Eingabefenster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>1107</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Eingabefenster</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Ei-O-Mat&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Eiergröße / Gewicht:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="ButtonSizeS">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Größe S</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ButtonSizeM">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Größe M</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ButtonSizeL">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Größe L</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ButtonSizeXL">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Größe XL</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Gewicht:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string> g</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="minimum">
             <number>43</number>
            </property>
            <property name="maximum">
             <number>83</number>
            </property>
            <property name="value">
             <number>58</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Temperatur des Ei:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioButtonKuehlschrank">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Kühlschrank</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonRaumtemperatur">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Raumtemperatur</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>benutzerdefinierte Temperatur</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxStarttemp">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string>  °C</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>40</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_hight">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Höhe über NN:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="altitude">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="suffix">
           <string> m ü. NN</string>
          </property>
          <property name="minimum">
           <number>-1000</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>115</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="currentText">
           <string>Karlsruhe (115 m ü. NN)</string>
          </property>
          <item>
           <property name="text">
            <string>Karlsruhe (115 m ü. NN)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stuttgart (250 m ü. NN)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Zugspitze (2.960 m ü. NN)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Benutzerdefiniert</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Zieltemperatur Eigelb: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_targettemp">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>550</width>
            <height>320</height>
           </size>
          </property>
          <widget class="QDial" name="dial">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>-20</y>
             <width>251</width>
             <height>311</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>300</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="tabletTracking">
            <bool>false</bool>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="minimum">
            <number>62</number>
           </property>
           <property name="maximum">
            <number>82</number>
           </property>
           <property name="pageStep">
            <number>2</number>
           </property>
           <property name="value">
            <number>72</number>
           </property>
           <property name="sliderPosition">
            <number>72</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>250</y>
             <width>141</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Flüssiges Eigelb</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>141</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Weiches Eigelb</string>
           </property>
          </widget>
          <widget class="Line" name="line_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>141</width>
             <height>20</height>
            </rect>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>410</x>
             <y>100</y>
             <width>141</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Hartes Eigelb</string>
           </property>
          </widget>
          <widget class="Line" name="line_3">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>240</y>
             <width>171</width>
             <height>20</height>
            </rect>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="Line" name="line_4">
           <property name="geometry">
            <rect>
             <x>400</x>
             <y>110</y>
             <width>131</width>
             <height>51</height>
            </rect>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Kochzeit:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="calculatedTime">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Timer Sound auswählen (.wav datei in QT PRojektordner):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>keine .wav Datei im QT Projektordner</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="start_Timer">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Timer starten</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Eier nach dem Kochen in kaltem Wasser abschrecken. </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
